### PR TITLE
Improving syntax for bash scripts

### PR DESCRIPTION
**Pull request by rlenferink**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Improves the syntax for the current bash scripts.

**Which issue(s) this PR fixes**:
#73082 is used for tracking the progress of all PRs.

**Special notes for your reviewer**:
None

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/assign @fejta @ixdy

### CODE REVIEW

**Comment by rlenferink** [comment]
/assign @BenTheElder 

Assigning you as well Ben, as I just noticed your comment on the issue and it's keep to get the changes aligned ;)

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -858,7 +858,7 @@ function start_kubelet {
       done
     fi
     # Quick check that kubelet is running.
-    if [ -n "$KUBELET_PID" ] && ps -p $KUBELET_PID > /dev/null; then
+    if [ -n "$KUBELET_PID" ] && ps -p ${KUBELET_PID} > /dev/null; then
```
Comment (ixdy) [inline hack/local-up-cluster.sh]: missed one here?

**Comment by BenTheElder** [comment]
thanks. agree with @ixdy's nit :-)

**Comment by fejta** [comment]
Ideally fix the nit before we manage to merge it. Otherwise another PR? And thanks!

File: hack/local-up-cluster.sh (hunk 1 above)
Comment (rlenferink) [reply hack/local-up-cluster.sh]: Whoops! Forgotten a few.
I've added them. But I try not to make the PR to big as it is more error prone when having to many changes at once.

**Comment by fejta** [comment]
@kubernetes/sig-testing 
/sig bash

**Comment by rlenferink** [comment]
> Ideally fix the nit before we manage to merge it. Otherwise another PR? And thanks!

Some more brackets have been added to `hack/local-up-cluster.sh`. Preferably, I'll just create a new PR with some follow-up improvements.

File: hack/local-up-cluster.sh (hunk 1 above)
Comment (ixdy) [reply hack/local-up-cluster.sh]: yeah, an iterative approach seems like a good idea. I mostly nitted here because it was on a line you already changed :)

**Pull request merged**